ParquetClassLibrary/EntityID.cs

```diff
@@ -91,6 +92,27 @@ public bool IsValidForRange(Range<EntityID> in_range)
             return _id == None || in_range.ContainsValue(Math.Abs(_id));
         }
 
+        /// <summary>
+        /// Validates the current ID over a collection of <see cref="Utilities.Range{EntityID}"/>s.
+        /// An ID is valid if:
+        /// 1) it is <see cref="Sandbox.ID.EntityID.None"/>
+        /// 2) it is defined within the given range, regardless of sign.
+        /// </summary>
+        /// <returns><c>true</c>, if the identifier is valid given any of the given ranges, <c>false</c> otherwise.</returns>
+        /// <param name="in_ranges">The range within which the absolute value of the ID must fall.</param>
+        [Pure]
```

Comment:
silly comment: I haven't seen [Pure] before, and now I'm thinking of various distracting jokes. If this ever gets removed, I can comment "It was too [Pure] for this world"

Reply:
YES!